shell/utils/array.ts

```diff
@@ -155,6 +155,27 @@ export function findBy<T, K, V>(
   return findOrFilterBy('find', ary, keyOrObj, val);
 }
 
+export function findStringIndex(items: string[], item: string, trim = true): number {
+  return items.indexOf(trim ? item?.trim() : item);
+}
+
+export function findStringDuplicates(items: string[], caseSensitive = true): boolean {
```

Comment:
I am very sorry to be this annoying, but since it's returning boolean and not a string, I would rather use `hasDuplicatedStrings` as name.